neuraxle/metaopt/trial.py

```diff
@@ -291,10 +490,10 @@ def get_best_hyperparams(self) -> HyperparameterSamples:
         best_score = None
         best_hyperparams = None
 
-        higher_score_is_better = self.trials[-1].metrics_results['main']['higher_score_is_better']
```

Comment:
This line often prints me an error when there are no trials existing. It would be worth to catch the error and to print a better error in that case here. Or else to avoid taking the [-1] thing when it doesn't exist.